plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/model/CubridUser.java

```diff
@@ -115,4 +157,140 @@ public List<CubridView> getSystemViews(DBRProgressMonitor monitor) throws DBExce
         }
         return views;
     }
+
+    @Nullable
+    @Override
+    public List<GenericTableIndex> getIndexes(@NotNull DBRProgressMonitor monitor) throws DBException {
+        cacheIndexes(monitor);
+        return cubridIndexCache.getAllObjects(monitor, this);
+    }
+
+    private void cacheIndexes(@NotNull DBRProgressMonitor monitor) throws DBException {
+        synchronized (cubridIndexCache) {
+            if(!cubridIndexCache.isFullyCached()) {
+                List<GenericTableIndex> newIndexCache = new ArrayList<>();
+                for(CubridTable table : getPhysicalTables(monitor)) {
+                    newIndexCache.addAll(table.getIndexes(monitor));
+                }
+                cubridIndexCache.setCache(newIndexCache);
+            }
+        }
+    }
+
+    public class CubridTableCache extends TableCache
+    {
+        protected CubridTableCache(@NotNull GenericDataSource dataSource) {
+            super(dataSource);
+        }
+
+        @NotNull
+        @Override
+        protected GenericTableColumn fetchChild(
+                @NotNull JDBCSession session,
+                @NotNull GenericStructContainer owner,
+                @NotNull GenericTableBase table,
+                @NotNull JDBCResultSet dbResult)
+                throws SQLException, DBException {
+            String columnName = JDBCUtils.safeGetString(dbResult, "attr_name");
+            String dataType = JDBCUtils.safeGetString(dbResult, "data_type");
+            Boolean autoIncrement = false;
+            String sql = "show columns from " + ((CubridDataSource) getDataSource()).getMetaModel().getTableOrViewName(table)
+                    + " where Field = ?";

```

Comment:
You need to quote object names in SQL. DBUtils.getQuotedIdentitier.

Reply:
I have added the DBUtils.getQuotedIdentitier.